ui/webpack.common.js

```diff
@@ -2,179 +2,198 @@ const path = require("path");
 const TsconfigPathsPlugin = require("tsconfig-paths-webpack-plugin");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 // webpack 5 stop handling node polyfills by itself, this plugin re-enables the feature
-const NodePolyfillPlugin = require("node-polyfill-webpack-plugin")
```

Comment:
I think this file looks ok, but all the formatting changes make it very hard to see the actual differences